### PR TITLE
Added meta and icons for item templates

### PR DESCRIPTION
**Pull request by davidortinau**
### Description of Change ###

Adds `ide.host.json` for each item template, and icons so they will appear in VS when you select to add an item.

### PR Checklist ###

- [x] Is Awesome

### CODE REVIEW

**Comment by phenning** [comment]
LGTM :shipit:

**Review by phenning — APPROVED** [review]

**Comment by davidortinau** [comment]
Looking now at adding post actions. I guess it's not required, but expected?

**Comment by phenning** [comment]
> Looking now at adding post actions. I guess it's not required, but expected?

Yes It's not needed for the basic scenario of having the templates appear., thus the ack. You should be able to just grab the primaryOutputs and postActions from the project templates and modify. 

If you do it in this PR, I'll take another 👀

File: src/Templates/src/templates/maui-contentpage-csharp/.template.config/ide.host.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+    "$schema": "http://json.schemastore.org/vs-2017.3.host",
+    "appliesTo": "UseMaui",
+    "defaultItemExtension": "cs",
+    "itemHierarchyPaths": [ ".NET MAUI" ],
+    "icon": "ide/MauiPageCSFile.ico"
+}
```
Comment (Eilon) [inline src/Templates/src/templates/maui-contentpage-csharp/.template.config/ide.host.json:7]:
```suggestion
}
```

Removing the niggling whitespace at the end 😁

File: src/Templates/src/templates/maui-contentpage-xaml/.template.config/ide.host.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
+    "$schema": "http://json.schemastore.org/vs-2017.3.host",
+    "appliesTo": "UseMaui",
+    "defaultItemExtension": "xaml",
+    "itemHierarchyPaths": [ ".NET MAUI" ],
+    "icon": "ide/MauiPageXamlFile.ico"
+}
```
Comment (Eilon) [inline src/Templates/src/templates/maui-contentpage-xaml/.template.config/ide.host.json:7]:
```suggestion
}
```

File: src/Templates/src/templates/maui-contentview-xaml/.template.config/ide.host.json [hunk 3]
```diff
@@ -0,0 +1,7 @@
+{
+    "$schema": "http://json.schemastore.org/vs-2017.3.host",
+    "appliesTo": "UseMaui",
+    "defaultItemExtension": "xaml",
+    "itemHierarchyPaths": [ ".NET MAUI" ],
+    "icon": "ide/MauiContentViewXamlFile.ico"
+}
```
Comment (Eilon) [inline src/Templates/src/templates/maui-contentview-xaml/.template.config/ide.host.json:7]:
```suggestion
}
```

**Review by Redth — APPROVED** [review]

**Pull request merged**